doc/quick-start-user-guide.md

```diff
@@ -39,19 +46,18 @@ Install packages that provide GlusterFS server (brick process) and client (fuse,
 
 As we do not have releases often, our nightly RPMs are generally more stable
 as they contain the latest fixes. If you are on centos 7, you can download the
-latest glusterd2 nightly RPM using the following method:
+latest glusterd2 nightly [RPMs](https://github.com/gluster/glusterd2/wiki/Nightly-Builds) using the following method:
 
 ```sh
 # curl -o /etc/yum.repos.d/glusterd2-nightly-master.repo http://artifacts.ci.centos.org/gluster/gd2-nightly/gd2-master.repo
 # yum install glusterd2
 ```
 
-Alternatlively, if you are using a non-RPM based distro, you can download
-binaries of the latest release. Like all Go programs, glusterd2 is a single
-binary (statically linked) without external dependencies. You can download the
-[latest release](https://github.com/gluster/glusterd2/releases) from Github.
+For other distros, you can download binaries of the [latest release](https://github.com/gluster/glusterd2/releases) from Github. 
```

Comment:
I really do not see how this change fixes or improves anything

Reply:
Fedora does not come under "non-rpm distros", but to install glusterd2 in Fedora we need to use binaries, so it is better to mention just distros to make it more generalized.